lib/workers/repository/update/pr/index.ts

```diff
@@ -281,9 +295,10 @@ export async function ensurePr(
   try {
     if (existingPr) {
       logger.debug('Processing existing PR');
+
       if (
         !existingPr.hasAssignees &&
-        !existingPr.hasReviewers &&
+        !hasNotIgnoredReviewers(existingPr, config) &&
```

Comment:
@rarkins it's now using the `reviewers` property directly, so no need for `hasReviewers` property